packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -436,8 +438,20 @@ export function TransferPanelMain({
     return result
   }, [erc20L1Balances, erc20L2Balances, selectedToken])
 
-  const externalFrom = isConnectedToArbitrum ? l2.network : l1.network
-  const externalTo = isConnectedToArbitrum ? l1.network : l2.network
+  const [externalFrom, externalTo] = useMemo(() => {
```

Comment:
Let me know if you need extra clarification, but we basically set default `from` and `to` based on different connected network combinations.